### PR TITLE
Update polaris breadcrumbs

### PR DESCRIPTION
**Pull request by tomnez**
v3.10 - v5.5 diff [here](https://github.com/Shopify/polaris-react/compare/v3.10.0...v5.5.0)

- [breaking] removes support for passing a custom class as argument (`@class`). Use HTML attributes instead with angle-bracket syntax
- converts to Glimmer component
- co-locates template
- misc 🧹

### CODE REVIEW

File: addon/components/polaris-breadcrumbs.js
Comment (vladucu) [inline addon/components/polaris-breadcrumbs.js:19]: wouldn't removing this default blow up if breadcrumbs is undefined and we assume it's an array?

File: addon/components/polaris-breadcrumbs.hbs
Comment (vladucu) [inline addon/components/polaris-breadcrumbs.hbs:16]:
```suggestion
          <PolarisIcon @source="ChevronLeftMinor" />
```

File: addon/components/polaris-breadcrumbs.hbs
Comment (vladucu) [inline addon/components/polaris-breadcrumbs.hbs:33]:
```suggestion
          <PolarisIcon @source="ChevronLeftMinor" />
```

File: addon/components/polaris-breadcrumbs.hbs
Comment (vladucu) [inline addon/components/polaris-breadcrumbs.hbs:1]: if there is no `breadcrumb`, we shouldn't be rendering anything. Can we do that here like [original version](https://github.com/Shopify/polaris-react/blob/master/src/components/Breadcrumbs/Breadcrumbs.tsx#L27) pls

**Review by vladucu — CHANGES_REQUESTED** [review]
Minor comments and we need to fix tests

File: addon/components/polaris-breadcrumbs.js
Comment (tomnez) [reply addon/components/polaris-breadcrumbs.js:19]: Not in this case since we only use it in `get breadcrumb()` which won't run unless `breadcrumbs` changes (a.k.a. gets passed in as a value). But to your point maybe we should keep it in case things change in the future?

Should we create a getter with `this.args.breadcrumbs || []`, or just do a check inside `get breadcrumb()` since that's the only spot it's used? Any strong opinion here? @vladucu

File: addon/components/polaris-breadcrumbs.hbs
Comment (vladucu) [inline addon/components/polaris-breadcrumbs.hbs:15]: @tomnez I've merged #678 which adds support for this new design language option.
Let's update this and remove the TODO before considering this done

File: addon/components/polaris-breadcrumbs.js
Comment (tomnez) [inline addon/components/polaris-breadcrumbs.js:16]: Will get this done before merging once we get my PolarisBadge PR merged.

**Review by vladucu — COMMENTED** [review]
@tomnez there are some merge conflicts here, let's get those addressed before another review pls

**Review by vladucu — CHANGES_REQUESTED** [review]
There are some failing specs related to changes here.

Note that [this](https://github.com/smile-io/ember-polaris/pull/677) was merged which is a component used by breadcrumbs.

**Review by vladucu — APPROVED** [review]
There's a pending TODO left, but otherwise this looks really nice.
Thanks for the updates, nicely done! 🚀

**Pull request merged**
